### DISCUSSION TITLE
Opt-in "Contact server admin" for troubleshooting

### DISCUSSION THREAD

**Discussion by theDavidCoen** [btcpayserver/btcpayserver#4806] [Ideas / Feature Requests]
I would suggest to add a "**Contact server admin**" link for problem with the servers, both in homepage and in the users' dashboard.

The server admin might activate this feature in Server Settings and enter an email address, to do support if needed.

This way, support request can be addressed directly to the server owner instead of the BTCpay Server team.

**Comment by dennisreimann** [comment]
Conceptually related to #4303, where we introduce this on a store-level. Makes sense to have it on the server-level as well.

Concept ACK!
